chaossearch/client/operation_create_object_group.go

```diff
@@ -50,6 +50,11 @@ func marshalCreateObjectGroupRequest(req *CreateObjectGroupRequest) ([]byte, err
 		},
 	}
 
+	if len(req.ColumnRenames) > 0 {
+		var options = body["options"].(map[string]interface{})
```

Comment:
I wanted to suggest to add a specific type for the ColumnRenamesRequest struct instead of this pretty generic type assertion, but then I saw that the whole body we construct is also a generic interface. So my question is, should we maybe on the line #35 instead of constructing a generic interface, have a CreateObjectGroupRequestBody structure or smth like this. Then if I understand correctly, we won't need this type assertion. WDYT?

Reply:
I'm not sure what the value of that is.

Furthermore, I'm not sure how to encode this kind of behavior without a custom marshaller:

```
	if req.Format == "LOG" {
		var format = body["format"].(map[string]interface{})
		format["pattern"] = req.Pattern
	}
```

and between a custom marshaller and a generic `map[string]interface{}` I think a map is simpler.